addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -68,6 +68,23 @@ def add_website_homepages(env):
     )
 
 
+def update_menu_urls(env):
+    """Update some menu URLs that must change."""
+    # Fix menus that include the "website." prefix
+    menus = env["website.menu"].search([
+        ("url", "=like", "/page/website.%"),
+    ])
+    for menu in menus:
+        menu.url = menu.url.replace("website.", "", 1)
+    # Fix menus pointing to "/page/homepage"
+    menus = env["website.menu"].search([
+        ("url", "=", "/page/homepage"),
```

Comment:
This must use `=like` the same as above as you can have anchors to homepage (`/page/homepage#anchor`).

Reply:
As talked through chat, we need to split this is 2 conditions.

Reply:
Fixed, thanks!